### PR TITLE
Address suggestions from Sonar

### LINKED ISSUE #106: Incorporate suggestions from Sonar
**Is your feature request related to a problem? Please describe.**
Initial Sonar scanning showed good code quality. However, there are some comments that need to be addressed.

**Describe the solution you'd like**
Refactor the code in order to sattisfy Sonar rules.

**Describe alternatives you've considered**
Eliminate meaningless suggestions from Sonar by modification of default set of rules

### PR DESCRIPTION
**Pull request by snk4tr**
Closes #106 

## Proposed Changes

  - Refactor some code blocks that marked by Sonar as smell
  - General refactoring for simplification of some logical statements
  - Reordering metrics and measures in the README.md alphabetically

### CODE REVIEW

**Comment by snk4tr** [comment]
@zakajd @denproc ready

File: piq/msid.py [hunk 1]
```diff
@@ -37,53 +37,49 @@ def _construct_graph_sparse(data: np.ndarray, k: int) -> np.ndarray:
...
+    return eye(matrix.shape[0]) - row_sum_sqrt.dot(matrix).dot(row_sum_sqrt)
 
 
-def _lanczos_m(
-        A: np.ndarray,
-        m: int,
-        nv: int,
-        rademacher: bool,
-        SV: np.ndarray = None) -> Tuple[np.ndarray, np.ndarray]:
+def _lanczos_m(A: np.ndarray, m: int, nv: int, rademacher: bool, starting_vectors: np.ndarray = None) \
```
Comment (denproc) [inline piq/msid.py:49]: SonarCloud smell

L139, L159, L180, L204, L286 too.

Comment (snk4tr) [reply piq/msid.py:49]: I did not change this one on purpose since it is really not clear how to call this `A` since it is much more than just a matrix

**Comment by snk4tr** [comment]
@denproc I propose to not fight for each and every tiny detail of Sonar satisfaction because it is going to take forever and might lead to even worse code quality than it is now

**Comment by snk4tr** [comment]
@zakajd fair point, totally forgot that. Reordered metrics as was described.
I also removed table of contents since it makes no sense

**Pull request merged**